PowerPointLabs/PowerPointLabs/SyncLab/SyncLabShapeStorage.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+
+using Microsoft.Office.Interop.PowerPoint;
+
+using PowerPointLabs.Models;
+
+namespace PowerPointLabs.SyncLab
+{
+    public class SyncLabShapeStorage : PowerPointPresentation
+    {
+#pragma warning disable 0618
```

Comment:
Since SyncPaneWPF inherits [UserControl](https://msdn.microsoft.com/en-us/library/system.windows.controls.usercontrol(v=vs.110).aspx) I believe it's able to use the action framework's [extension methods](https://github.com/PowerPointLabs/PowerPointLabs/blob/master/PowerPointLabs/PowerPointLabs/ActionFramework/Common/Extension/ContentControlExtensions.cs). for example, like [this](https://github.com/PowerPointLabs/PowerPointLabs/blob/master/PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs#L261). fyi, [what is extension method](https://msdn.microsoft.com/en-us//library/bb383977.aspx).